tangelo/algorithms/classical/fci_solver.py

```diff
@@ -210,10 +214,19 @@ def simulate(self):
             for swap_op in swap_ops:
                 wfn.Ca().rotate_columns(0, swap_op[0], swap_op[1], np.deg2rad(90))
 
+        if hasattr(self, "chrgfield"):
```

Comment:
Let's leave a comment about the versions here. I don't know how recent is v1.6 but eventually there will be a todo here and the branch for older versions will be pruned.